automol/util/vector.py

```diff
@@ -112,7 +110,7 @@ def best_unit_perpendicular(xyzs):
     Solution to the last problem was found here: https://stackoverflow.com/a/51132260
 
     :param xyzs: The points
-    :type xyzs: List[Vector]
+    :return: New vector
     """
```

Comment:
Here, we will need to require all of the elements of the input list to be vectors, which can be done as follows:
```
assert all(is_vector_like(xyz) for xyz in xyzs), f"One of these is not vector like: {xyzs}"
```